toolkit/docs/nvidia/mariner-nvidia.repo

```diff
@@ -0,0 +1,9 @@
+[mariner-official-nvidia]
```

Comment:
Why not make this a subpackage for `mariner-repos`?

Reply:
because we want this to be a manual process by design, as I mentioned last week.